build/dev/startup/scd_bootstrapper.sh

```diff
@@ -3,6 +3,8 @@
 # This startup script is meant to be invoked from within a Docker container
 # started by docker-compose_dss.yaml, not on a local system.
 
+# TODO: Is the test disabled on purpose?
+# shellcheck disable=SC2157
```

Comment:
I think this is also a bug -- we want to execute the command in that string, not check if it's non-empty :)